db/schema.rb

```diff
@@ -10,7 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_04_15_143554) do
+ActiveRecord::Schema[7.1].define(version: 2024_06_11_092435) do
+  create_schema "analytics"
```

Comment:
Why do we have an additional schema?

Reply:
I believe this creeped in from something David may have tried previously. will take a look at removing or using